### DISCUSSION TITLE
Install/Setup Problem "No database connection" - Port Setting?

### DISCUSSION THREAD

**Discussion by trist** [Leantime/leantime#681] [General]
Trying to setup Leantime using Docker image (on Synology NAS). 

My MariaDB container can't use the standard port. Is there a way to set leantime's db port env? I've tried `LEAN_DB_PORT` but still get message in browser: "No database connection, check your database credentials in your configuration file."

**Comment by marcelfolaron** [comment]
Yes, just add the port number to the host using a colon: domain:3306

I have the same set up on my synology.

**Reply by trist** [reply]
Thanks for reply. I did try that with no result. I'll give it another Go and post back when I get success

**Comment by marcelfolaron** [comment]
Also, make sure to call the right IP address. Host names didn’t seem to
work for me.

**Reply by hifron** [reply]
On my Windows I have to setup file C:\Windows\System32\drivers\etc\hosts to have something like 

```
127.0.0.1       localhost leantime.localhost blog.localhost
	::1             localhost leantime.localhost blog.localhost
```

and in Apache conf/extra/httpd-vhosts.conf
```
<VirtualHost *:80>

    ServerAdmin my@email.com

    DocumentRoot "C:/Users/user_name/Projects/LeanTime/"

    ServerName leantime.localhost

    ErrorLog "logs/leantime-error.log"

    CustomLog "logs/leantime-access.log" common

</VirtualHost>
```

and also setup main Apache conf\httpd.conf 

```
<Directory />
    AllowOverride All

    Require all denied

</Directory>

```
```
<Directory "C:/Users/user_name/Projects">

    Options Indexes FollowSymLinks Includes ExecCGI

    DirectoryIndex index.php index.html

    AllowOverride All

    Require all granted
</Directory>

```

Also there is required to update public/.htaccess file like in [Issue 617](https://github.com/Leantime/leantime/issues/617)

After that leantime.localhost would be(after firewall popup) accessible on leantime.pc-name on local network from Windows machine(and maybe some other firewall settings to allow ping pc-name).


On Linux there should be something similar like /etc/something..., but probably needed some Avahi, winbind installing and setup, including firewall settings which is not so trivial...

**Comment by Emad-Marji** [comment]
Hi @marcelfolaron,
It wasn't easy to install it with container manager in DSM 7.2; but finally managed :)
It seems to be working fine under the localhost, but when using the reverse proxy in Synology, i can't access the dashboard. When accessing using the localhost, it is there, but when using my local domain I can access everything except the dashboard!!!
any advice? do i need to configure the domain in the .env file?
